### PR TITLE
layers: Fix 02679 validation

### PR DESCRIPTION
**Pull request by arno-lunarg**
- Closes https://github.com/KhronosGroup/Vulkan-ValidationLayers/issues/5938

Also add a regression test

### CODE REVIEW

**Review by spencer-lunarg — APPROVED** [review]

**Review by juan-lunarg — CHANGES_REQUESTED** [review]
The issue (https://github.com/KhronosGroup/Vulkan-ValidationLayers/issues/5938) in question points to a false positive. Shouldn't a positive test be added?

**Comment by arno-lunarg** [comment]
@juan-lunarg  The last few lines of the new test is the positive part, but I can just add a proper positive test if you want

File: tests/negative/wsi.cpp [hunk 1]
```diff
@@ -2855,6 +2855,60 @@ TEST_F(NegativeWsi, AcquireFullScreenExclusiveModeEXT) {
...
+    VkSwapchainKHR swapchain = VK_NULL_HANDLE;
+
+    surface_full_screen_exlusive_info.fullScreenExclusive = VK_FULL_SCREEN_EXCLUSIVE_APPLICATION_CONTROLLED_EXT;
+    m_errorMonitor->SetDesiredFailureMsg(kErrorBit, "VUID-VkSwapchainCreateInfoKHR-pNext-02679");
+    vk::CreateSwapchainKHR(device(), &swapchain_create_info, nullptr, &swapchain);
+    m_errorMonitor->VerifyFound();
+
+    surface_full_screen_exlusive_info.fullScreenExclusive = VK_FULL_SCREEN_EXCLUSIVE_DEFAULT_EXT;
+    vk::CreateSwapchainKHR(device(), &swapchain_create_info, nullptr, &swapchain);
+    vk::DestroySwapchainKHR(device(), swapchain, nullptr);
```
Comment (juan-lunarg) [inline tests/negative/wsi.cpp:2908]: > @juan-lunarg The last few lines of the new test is the positive part, but I can just add a proper positive test if you want

A separate positive test would be my preference. Or adding a comment. My main nitpick is it's not immediately obvious that positive behavior is being tested.

It's obvious from the context of this PR. But it's not as obvious for a future test maintainer.

Comment (arno-lunarg) [reply tests/negative/wsi.cpp:2908]: True, I am going to add a proper positive test

**Review by juan-lunarg — APPROVED** [review]

**Pull request merged**